From bc858640e9cba5b7b1a657ad6ab7469b82dfb556 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Thu, 16 Jan 2025 14:14:08 +0100
Subject: [PATCH] pam_systemd_home: use right field name in error message

(cherry picked from commit 1fb53bb561db11e72bb695d578f8e94042565822)

Resolves: RHEL-109902
---
 src/home/pam_systemd_home.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/home/pam_systemd_home.c b/src/home/pam_systemd_home.c
index fb61105295..794438129e 100644
--- a/src/home/pam_systemd_home.c
+++ b/src/home/pam_systemd_home.c
@@ -202,7 +202,7 @@ static int acquire_user_record(
                 r = pam_set_data(handle, generic_field, json_copy, pam_cleanup_free);
                 if (r != PAM_SUCCESS)
                         return pam_syslog_pam_error(handle, LOG_ERR, r,
-                                                    "Failed to set PAM user record data '%s': @PAMERR@", homed_field);
+                                                    "Failed to set PAM user record data '%s': @PAMERR@", generic_field);
 
                 TAKE_PTR(json_copy);
         }
